tools/MVP/templates/view.py.in

```diff
@@ -32,4 +30,4 @@ class View(QWidget):
         self._presenter.handle_button_clicked()
 
     def set_label(self, text: str) -> None:
-        self._label.setText(text)
+        self.lbNumberInc.setText(text)
```

Comment:
This is just a minor suggestion, feel free to ignore - the `lbNumberInc` widget we could name `lbCount` so that it corresponds better with the "_count" variable in the Model